SPECS/abseil-cpp/abseil-cpp.spec

```diff
@@ -0,0 +1,117 @@
+Summary:        C++ Common Libraries
+Name:           abseil-cpp
+Version:        20211102.0
+Release:        1%{?dist}
+License:        ASL 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://abseil.io
+Source0:        https://github.com/abseil/abseil-cpp/archive/%{version}/%{name}-%{version}.tar.gz
+# Workaround until GTest publishes a release including the "GTEST_FLAG_GET" macro.
+# Currently only available in the "main" branch: https://github.com/google/googletest/commit/977cffc4423a2d6c0df3fc9a7b5253b8f79c3f18
+Patch0:         gtest_build_fix.patch
+# Workaround until GTest publishes a release including the "::testing::Conditional" matcher.
+# Currently only available in the "main" branch: https://github.com/google/googletest/commit/8306020a3e9eceafec65508868d7ab5c63bb41f7
+Patch1:         disabling_invalid_tests.patch
+
+BuildRequires:  cmake >= 3.20.0
+BuildRequires:  gcc
```

Comment:
nit: these are always available in the build chroot, so it's not necessary to include them as BR

Reply:
There's no harm in keeping them there and being explicit about build dependencies seems like a good idea, IMO. Although I'm aware we never really capture all of the most basic ones. And I think `cmake` is part of the toolchain but not part of the chroot.